common/app/model/LiveBlogCurrentPage.scala

```diff
@@ -70,6 +74,18 @@ object LiveBlogCurrentPage {
         BlockPage(blocks = Nil, blockId = blockId, pageNumber = numPages, filterKeyEvents)
       }
 
+      // returns latest pinned post and blocks minus the pinned post
+      val (pinnedPost, filteredFirstPageBlocks) = {
+        val pinnedPosts = blocks.requestedBodyBlocks.get(CanonicalLiveBlog.pinned)
+        val pinnedPost = pinnedPosts.flatMap(_.headOption)
+        val pinnedPostId = pinnedPost.map(_.id).getOrElse("")
+        val firstBlockId = firstPageBlocks.headOption.map(_.id).getOrElse("")
+        val filteredFirstPageBlocks =
+          firstPageBlocks.filter(block => !(block.id == pinnedPostId && block.id == firstBlockId))
```

Comment:
I think that would lead to a different outcome ie:
```(!foo && !bar) is not the same as !(foo && bar)```
also see Alina's suggestion of using `filterNot` above